packages/next-templates/src/app/meldingen-openbare-ruimte/graffiti-melding/form/page.tsx

```diff
@@ -130,6 +126,7 @@ export default function Home() {
                     required: 'Dit is verplicht',
                     minLength: { value: 4, message: 'Min length is 4' },
                   })}
+                  name="Omschrijving"
```

Comment:
name wilt je in het engels want die naam gebruikt de backend. dus je wilt het consistent houden omdat de id en classname ook engels zijn.